RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETField.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace RepositoryCompiler.CodeModel.CaDETModel.CodeItems
+{
+    public class CaDETField
+    {
+        public string Name { get; internal set; }
+        public List<CaDETModifier> Modifiers { get; internal set; }
```

Comment:
What does it represent field "Modifiers" ?

Reply:
Modifiers are keywords that change the behavior/meaning of a function, field, or class. These include "static", "abstract", and visibility modifiers such as private, public, internal...
Currently we do not utilize them in our smell detectors, but it might be interesting to see if a class has for example 10+ public methods and few private methods - signaling that it might have too many responsibilities.